internal/apiserver/route_get_identity_by_id.go

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http"
+
+	"github.com/hyperledger/firefly/internal/config"
+	"github.com/hyperledger/firefly/internal/i18n"
+	"github.com/hyperledger/firefly/internal/oapispec"
+	"github.com/hyperledger/firefly/pkg/fftypes"
+)
+
+var getIdentityByID = &oapispec.Route{
+	Name:   "getIdentityByID",
+	Path:   "namespaces/{ns}/identities/{idid}",
```

Comment:
minor thing, but `idid` does look a bit silly 😅 and also has DID in the name despite not being a DID endpoint. Would just `id` or `iid` be any better?

Reply:
Fixed this 👍 

Also added `/api/v1/verifiers` resource collection, because I realized there was no API for reverse lookups any more without that.